BGL/include/CGAL/boost/graph/Euler_operations.h

```diff
@@ -1021,21 +1021,21 @@ add_face_to_border(typename boost::graph_traits<Graph>::halfedge_descriptor h1,
  *
  * \tparam Graph must be a model of `MutableFaceGraph`
  * Let `h` be the halfedge of `e`, and let `v0` and `v1` be the source and target vertices of `h`.
- * Let `ep` and `e'p` be respectively the edges of `prev(h,g)` and `prev(opposite(h, g), g)`.
+ * Let `hp` and `e'p` be respectively the edges of `prev(h,g)` and `prev(opposite(h, g), g)`.
```

Comment:
Imho that's not any better... Now we have `h` and `e` mixed...
In the first line, "prime" indicates that we are taking the opposite, and in the second line, it does not have the same meaning

Reply:
so hp and h'p ?